services/frontend-v3/src/routes/Routes.jsx

```diff
@@ -33,7 +38,12 @@ const Routes = () => (
         render={({ location }) => <Secured location={location} />}
       />
     </Switch>
+    <PrivacyModal />
   </>
-);
+)
+
+
+
```

Comment:
Probably want to remove the superfluous lines/spaces

Reply:
removed